src/main/kotlin/blackjack/domain/BlackJackCards.kt

```diff
@@ -0,0 +1,38 @@
+package blackjack.domain
+
+class BlackJackCards(private val _cards: MutableList<BlackJackCard> = mutableListOf()) {
+    val cards: List<BlackJackCard>
+        get() {
+            return _cards.toList()
+        }
+
+    fun addCard(card: BlackJackCard) {
+        require(isCardNumberSumUnderBlackJackWinCardSum()) { "21을 넘으면 카드를 못뽑아요" }
+        _cards.add(card)
+    }
+
+    fun isCardNumberSumUnderBlackJackWinCardSum(): Boolean {
+        val bestSum = getCardsBestSum()
+        return (bestSum in 1..BLACKJACK_WIN_CARD_SUM)
```

Comment:
bestSum이 0이하가 나올 수 없고, 숫자 비교니까 아래처럼해도 괜찮지 않을까요? 

```suggestion
        return bestSum <= BLACKJACK_WIN_CARD_SUM
```

Reply:
22가 넘는 경우 -1를 리턴하도록 구현하였기 때문에 양수 처리가 필요합니다